nixos/modules/services/web-servers/h2o/default.nix

```diff
@@ -278,6 +284,16 @@ in
           '';
       };
 
+      defaultListenHost = mkOption {
+        type = types.nullOr types.nonEmptyStr;
+        default = null;
+        example = "0.0.0.0";
+        description = ''
+          Adds this to all listen.host blocks by default. Useful for using
+          `0.0.0.0` to expose all virtual hosts to the internet.
```

Comment:
This probably only exposes it to a small fraction of the internet. Let's suggest a more inclusive variant.

Reply:
Ugh. Normally `::` binds on both v4 and V6, but maybe `[::]` might be needed. It would help a lot to link to the docs of the directive we're setting here.

Reply:
0.0.0.0 will normally only bind on ipv4, even if the machine is dualstacked.